src/cluster/src/uninstall.rs

```diff
@@ -0,0 +1,259 @@
+    pub fn uninstall_sys(&self) -> Result<(), ClusterError> {
+        info!("Removing fluvio sys chart");
+        self.helm_client.uninstall(DEFAULT_CHART_SYS_REPO)?;
+        info!("fluvio sys chart has been uninstalled");
+        self.cleanup()?;
+
+        Ok(())
+    }
+
+    /// Uninstall local fluvio system chart
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    /// # use fluvio_cluster::ClusterUninstaller;
+    /// let uninstaller = ClusterUninstaller::new()
+    ///     .with_name("my-namespace")
+    ///     .build();
+    /// uninstaller.uninstall_local();
+    /// ```
+    pub fn uninstall_local(&self) -> Result<(), ClusterError> {
+        info!("Removing local cluster");
+        Command::new("pkill").arg("-f").arg("fluvio run").output()?;
+        // delete fluvio file
+        debug!("remove fluvio directory");
+        if let Err(err) = remove_dir_all("/tmp/fluvio") {
+            warn!("fluvio dir can't be removed: {}", err);
+        }
+        self.cleanup()?;
+        Ok(())
+    }
+
+    /// Clean up objects and secrets created during the installation process
+    fn cleanup(&self) -> Result<(), ClusterError> {
+        let ns = &self.config.namespace;
+
+        // delete objects
+        self.remove_objects("spugroups", ns, None)?;
+        self.remove_objects("spus", ns, None)?;
+        self.remove_objects("topics", ns, None)?;
+        self.remove_objects("persistentvolumeclaims", ns, Some("app=spu"))?;
+
+        // delete secrets
+        self.remove_secrets("fluvio-ca")?;
+        self.remove_secrets("fluvio-tls")?;
+
+        Ok(())
+    }
+
+    /// Remove objects of specified type, namespace
+    fn remove_objects(
+        &self,
+        object_type: &str,
+        namespace: &str,
+        selector: Option<&str>,
+    ) -> Result<(), ClusterError> {
+        let mut cmd = Command::new("kubectl");
+        cmd.arg("delete");
+        cmd.arg(object_type);
+        cmd.arg("--namespace");
+        cmd.arg(namespace);
+        if let Some(label) = selector {
+            info!(
+                "deleting label '{}' object {} in: {}",
+                label, object_type, namespace
+            );
+            cmd.arg("--selector").arg(label);
+        } else {
+            info!("deleting all {} in: {}", object_type, namespace);
+            cmd.arg("--all");
+        }
+        cmd.output()?;
+
+        Ok(())
+    }
+
+    /// Remove K8 secret
+    fn remove_secrets(&self, name: &str) -> Result<(), ClusterError> {
+        Command::new("kubectl")
+            .arg("delete")
+            .arg("secret")
+            .arg(name)
+            .arg("--ignore-not-found=true")
+            .output()?;
+
+        Ok(())
+    }
+
+    async fn wait_for_delete<S: Spec>(
+        &self,
+        client: SharedK8Client,
+        input: &InputObjectMeta,
+    ) -> Result<(), ClusterError> {
+        use k8_client::metadata::MetadataClient;
+        let mut success = false;
+        for i in 0..100u16 {
```

Comment:
can you make this configurable?

Reply:
should I use this `no_wait` option? https://github.com/infinyon/fluvio/blob/master/src/cli/src/cluster/uninstall.rs#L17

Reply:
No

Reply:
Okay do you want me to make the counter `i` as configurable?

Reply:
Yes